### PR TITLE
add Django Wireframe. Need to add HTML template and Data Models

### PR DESCRIPTION
**Pull request by Brahmancer**

### CODE REVIEW

File: FPMS/FPMS/settings.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 't+^flxg&v596q&3n_c(#kz5&k9*symytm!-qzd9#-o85s99yrz'
```
Comment (Cphrampus) [inline FPMS/FPMS/settings.py:23]: I realize this is for dev, but we need to set up some kind of clean/smudge filter pair or env var setup to get this from

**Review by Cphrampus — APPROVED** [review]

File: FPMS/FPMS/settings.py (hunk 1 above)
Comment (Brahmancer) [reply FPMS/FPMS/settings.py:23]: This is true. I didn't even see this. I'll see if I can filter this out

Comment (Cphrampus) [reply FPMS/FPMS/settings.py:23]: Like I said, it's probably fine for dev, but we can replace it with something like "<SECRET_TOKEN>" and have clean/smudge filters set up for us to switch it. They wouldn't be too hard to write, I can make a task for it and get a mock scripted, if that's the direction we want to go

Comment (Brahmancer) [reply FPMS/FPMS/settings.py:23]: Make it so.

**Pull request merged**